### ISSUE TITLE
feat: simplify exceptions handling API

### ISSUE THREAD

**Issue by uniqueg** [elixir-cloud-aai/foca#240]
The [`ExceptionConfig()` model/API](https://github.com/elixir-cloud-aai/foca/blob/f8699377c862d3d5479652207ec5f0419bf987a6/foca/models/config.py#L192-L440) is unnecessarily complicated.

Simplify with a focus on usability.

**Comment by kushagra189** [comment]
Any suggestions here?

**Comment by uniqueg** [comment]
Yes, I'm already working on it (I will describe it in the PR once it's done, but will take some weeks). Upgrading to Connexion 3 will require this change anyway.
